command/job_start.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package command
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/api/contexts"
+	"github.com/posener/complete"
+)
+
+type JobStartCommand struct {
+	Meta
+}
+
+func (c *JobStartCommand) Help() string {
+	helpText := `
+Usage: nomad job start [options] <job>
+Alias: nomad start
+
+
+ Starts a stopped job. The job must be currently registered with Nomad. Upon
+ successful start, an interactive monitor session will start to display log
+ lines as the job starts its allocations based on its most recent version.
+ It is safe to exit the monitor early using ctrl+c.
+
```

Comment:
I want to make explicit what Nomad is doing here in terms of job versions, as that seems to be a source of confusion for users. I've lifted some of this text from `job run`:

```suggestion
Starts a stopped job. The job must be currently registered. Nomad will create a
new version of the job based on its most recent version. Upon successful start,
Nomad will enter an interactive monitor session. This is useful to watch
Nomad's internals make scheduling decisions and place the submitted work onto
nodes. The monitor will end once job placement is done. It is safe to exit the
monitor early using ctrl+c.
```

(We'll want this to be in the docs too.)